osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -34,7 +35,24 @@ private void load(AudioManager audio)
             this.audio = audio;
         }
 
+        private bool looping;
+
+        public bool Looping
+        {
+            get => looping;
+            set
+            {
+                looping = value;
+
+                channels?.ForEach(c => c.Looping = looping);
+            }
+        }
+
         public void Play() => channels?.ForEach(c => c.Play());
+        public void Stop() => channels?.ForEach(c => c.Stop());
+
+        public void AddAdjustment(AdjustableProperty type, BindableDouble adjustBindable) => channels?.ForEach(c => c.AddAdjustment(type, adjustBindable));
```

Comment:
adjustments are lost after skin change, no?

Reply:
Fixed, now using a tuple list for taking care of adjusting new channels